code/game/objects/items/radio/radio.dm

```diff
@@ -253,6 +253,10 @@
 	if(!talking_movable.IsVocal())
 		return
 
+	var/mob/living/carbon/human/talker = talking_movable
+	if(isfallenjob(talker.job)) // prevents any of the valhalla jobs from talking to the living
+		return
```

Comment:
yeah this is omega snowflake, don't do this.

Reply:
Would it be better to make it var/mob/living/talker, or var/mob/living/carbon/talker? Unless you're specifically talking about the isfallenjob helper.

Reply:
If you don't want people in valhalla talking on radio, don't give them radios.